behave/features/constants.py

```diff
@@ -33,9 +33,16 @@
                                "login_url": f"{CLA_BACKEND_URL}/admin/login",
                                "user_type": "OPERATOR",
                                "application": "BACKEND"},
+    "FOX_ADMIN_NEW_USER": {"username": "elvis.presley",
+                           "password": "rockandroll",
+                           "login_url": f"{CLA_BACKEND_URL}/admin/login",
+                           "user_type": "OPERATOR",
+                           "application": "BACKEND"},
 }
-
-FOX_ADMIN_NEW_OPERATOR = "elvis.presley"
+# value_key is linked to FOX_ADMIN_NEW_USER
+FOX_ADMIN_NEW_OPERATOR = {"username": {"label": "Username:", "value_key": "username"},
```

Comment:
Is this used to describe the form fields when creating a new operator? If so what do you think of adding FORM_FIELDS or something similar to constant name?

Reply:
Cool, I like it. I've added `FOX_ADMIN_FORM_FIELDS`, sound good?